Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -1448,10 +1453,11 @@ namespace AtomSampleViewer
     {
         Script_SetShowImGui(false);
 
-        auto operation = [filePath]()
+        const AZStd::string& envPath = s_instance->m_envPath;
+        auto operation = [filePath, envPath]()
         {
             // Note this will pause the script until the capture is complete
-            if (PrepareForScreenCapture(filePath))
+            if (PrepareForScreenCapture(filePath, envPath))
```

Comment:
Wouldn't s_instance be accessible inside the lambda since it's a static variable?
envPath is a reference so there is no protection across threads reading & writing. Is there another reason to manually capture it?

Reply:
Yes, you are right. I'll update.